receiver/awsxrayreceiver/internal/translator/addtoattrs.go

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package translator
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+)
+
+const (
+	// AWSXRayInProgressAttribute is the `in_progress` flag in an X-Ray segment
+	AWSXRayInProgressAttribute = "aws.xray.inprogress"
```

Comment:
Similary, I think we'll need custom x-ray attributes to be shared between receiver and exporter and should go in the shared package